.github/workflows/lambdachecks.yml

```diff
@@ -34,4 +34,4 @@ jobs:
           ./aws/logs_monitoring/trace_forwarder/scripts/run_tests.sh
       - name: Run enhanced metric unittest
         run: |
-          python -m unittest
+          python -m unittest discover ./aws/logs_monitoring/
```

Comment:
Looks like you need to `pip install boto3` before running the tests.

Reply:
Yep, added now